pkg/proc/proc_test.go

```diff
@@ -3324,6 +3324,11 @@ func logStacktrace(t *testing.T, p *proc.Target, frames []proc.Stackframe) {
 			name = fmt.Sprintf("%s inlined in %s", frames[j].Call.Fn.Name, frames[j].Current.Fn.Name)
 		}
 
+		t.Logf("\t%#x %#x %#x %s at %s:%d\n",
```

Comment:
I think logging the stack trace twice is one too much. Motivation?

Reply:
Probably a rebase error but I agree this should be removed.